app/Http/Controllers/Register/regController.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace App\Http\Controllers\Register;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+use App\Models\Questions;
+use App\Models\CorrectAnswers;
+
+class regController extends Controller
+{
+    public function showForm()
+    {
+        return view('list.reg');
+    }
+    
+    public function validateForm(Request $request)
+    {
+        $question = $request['question'];
+        $answers = $request['answers'];
+        $status = true;
+        $error = $this->validation($question, $answers);
+        if ($error == null) {
+            return view('list.regConfirm', compact('question', 'answers', 'status'));
+        } else {
+            return view('list.reg', ['error' => $error]);
+        }
+    }
+    private function validation($question, $answers)
+    {
+        if($question == null || $answers == null){
+           return "未入力項目があります。"; 
+        }
+        if(mb_strlen($question) > 500){
+            return "問題は500文字以内で入力してください。";
+        }
+        for($i = 0; $i < count($answers); $i++){
```

Comment:
今日の感じだとここのforがちゃんと通るか怪しいですね…
```suggestion
        foreach ($answers as $answer){
          if(mb_strlen($answer) > 200){
```
にした方が良さそう。